oracle/src/RoflUtilityAppd.py

```diff
@@ -80,7 +80,7 @@ def submit_tx(self, tx: TxParams) -> typing.Any:
                 "kind": "eth",
                 "data": {
                     "gas_limit": tx["gas"],
-                    "value": tx["value"],
+                    "value": hex(tx["value"]),
```

Comment:
```suggestion
                    "value": str(tx["value"]),
```

I think that should be same as in the rofl client
https://github.com/oasisprotocol/oasis-sdk/blob/809273c3e70545a29818404a5076e91bde8455ce/rofl-client/py/src/oasis_rofl_client/rofl_client.py#L150

and maybe also correct the below to
```py
            "encrypt": false,
```

Reply:
```
"value": str(tx["value"]),
```

:+1: 

```
"encrypt": false,
```
^^ this is python's syntax, so `False`.